catalog/security/banner-marking/src/main/java/org/codice/alliance/security/banner/marking/BannerCommonMarkingExtractor.java

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.alliance.security.banner.marking;
+
+import static org.codice.alliance.security.banner.marking.BannerMarkings.MarkingType.FGI;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiFunction;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang.StringUtils;
+
+import com.google.common.collect.ImmutableList;
+
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.AttributeImpl;
+
+public class BannerCommonMarkingExtractor extends MarkingExtractor {
+    public BannerCommonMarkingExtractor() {
+        Map<String, BiFunction<Metacard, BannerMarkings, Attribute>> tempMap = new HashMap<>();
+
+        tempMap.put("security.classification",
```

Comment:
I would also recommend refactoring these strings into constants so that it's easier to go through and update when the taxonomy work completes.